lib/dialects/mssql/index.js

```diff
@@ -62,6 +62,7 @@ class Client_MSSQL extends Client {
         useColumnNames: false,
         tdsVersion: settings.options.tdsVersion || '7_4',
         appName: settings.options.appName || 'knex',
+        trustServerCertificate: false,
```

Comment:
Could you please remove the default value? Considering that it might silently change behaviour of currently running applications, that would be a breaking change.

Reply:
probably `false` default is fine here? I mean if that option has not been given earlier it couldn't have been true.

Reply:
I checked mssql documentation and you are right, false is the default